tests_func/Dockerfile

```diff
@@ -18,19 +19,22 @@ RUN apt-get update -qq && \
     apt-get install --allow-unauthenticated -y \
         ${MONGO_PACKAGE}-server=$MONGO_VERSION \
         ${MONGO_PACKAGE}-shell=$MONGO_VERSION \
-        ${MONGO_PACKAGE}-tools=$MONGO_VERSION \
-        pv && \
+        ${MONGO_PACKAGE}-tools=$MONGO_VERSION && \
     rm -rf /var/lib/apt/lists/* /var/cache/debconf /var/lib/mongodb/* && \
     apt-get clean
 
 RUN mkdir -p /home/mongodb /var/log/wal-g/ ${TMP_DIR} && \
     chown mongodb:mongodb /home/mongodb /var/log/wal-g/ && \
     ln --force -s /config/supervisor/conf.d/mongodb.conf /etc/supervisor/conf.d/mongodb.conf && \
-    mkdir -p ${WALG_REPO}
+    mkdir -p ${WALG_REPO} ${WALG_CONF_DIR} && \
+    mkdir -p /home/mongodb/.gnupg && touch /home/mongodb/.gnupg/gpg.conf && \
+    chmod -R 700 /home/mongodb/.gnupg && chown mongodb:mongodb -R /home/mongodb/.gnupg && \
+    ln --force -s /config/wal-g.json ${WALG_CONF_DIR}/wal-g.json && \
+    ln --force -s /config/gpg-key.armor ${WALG_CONF_DIR}/gpg-key.armor
 
 COPY wal-g ${WALG_REPO}/
 WORKDIR ${WALG_REPO}
-RUN make install && \
-    make deps && \
-    make mongo_build && \
-    cp main/mongo/wal-g /usr/bin/wal-g
\ No newline at end of file
+
+RUN make unlink_brotli && make unlink_libsodium && make link_brotli && make link_libsodium
+RUN make mongo_build && \
```

Comment:
So, after adding these tests to run full time of running takes almost 50 minutes. If it will exceed 50 minutes tests will start flap. We should think how to resolve it.